build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
 }
 
 repositories {
+    maven { url 'https://jitpack.io' }
```

Comment:
what needed jitpack? In general I'm not big on jitpack for upstream dependencies. AFAIK jitpack builds from source and goes down if github is down, similar it does not have the same security guarantees something like maven central or google repo has. Can we shade (include the jar in lib) whatever the dependecy is? I can also try to find a different one that fulfills same use case OR I can repackage under my own domain and throw on maven central.

Reply:
Yeah; I think what I'm going to end up doing instead is just including a built-jar inside this repo. kotlinx.ast isn't published anywhere else.